tests/imgtype.go

```diff
@@ -111,22 +112,22 @@ func main() {
 			ref = ref2
 		}
 
-		img, err := ref.NewImage(systemContext)
+		img, err := ref.NewImage(context.TODO(), systemContext)
```

Comment:
(In tests it’s OK to call `context.Background()`, i.e. without having them in the “to be fixed later” list, maybe unless you are planning to actually test the cancelation mechanism.)

Reply:
calling context.Background() now.